packages/jsii-python-runtime/tests/test_compliance.py

```diff
@@ -193,8 +193,8 @@ def test_primitiveTypes():
     assert types.date_property == datetime.fromtimestamp(123 / 1000.0, tz=timezone.utc)
 
     # json
-    types.json_property = {"Foo": 123}
-    assert types.json_property.get("Foo") == 123
+    types.json_property = { "Foo": { "bar": 123  } }
```

Comment:
Would it make sense to do that for java and .net as well?

Reply:
Did you verify that this indeed fails before the fix is applied?

(Sorry being a bit risk averse)

Reply:
Oh Java is going to be such fun to write!

Reply:
Nevermind... Java actually _prohibits_ this particular use-case through static typing!